### ISSUE TITLE
For accessibility on Mac OS, add "Increase list item indent" keyboard shortcut

### ISSUE THREAD

**Issue by co-operate** [ckeditor/ckeditor5#16964]
As per the [accessibility support docs](https://ckeditor.com/docs/ckeditor5/latest/features/accessibility.html#keystrokes-that-can-be-used-in-a-list), the "Increase list item indent" shortcut for Mac is missing, although one exists for Windows, and although a "Decrease indent" shortcut for Mac exists. I'd propose Option-Tab

---

If you'd like to see this feature implemented, add a 👍 reaction to this post.
